doc/classes/Array.xml

```diff
@@ -441,12 +441,32 @@
 			<return type="Variant" />
 			<description>
 				Returns the maximum value contained in the array if all elements are of comparable types. If the elements can't be compared, [code]null[/code] is returned.
+				To find the minimum value using a custom comparator, use [method reduce] like this (this returns the first maximum value if multiple values would be considered the maximum, as in the example):
+				[codeblock]
+				func _ready():
+				    var arr = [Vector2(0, 1), Vector2(2, 0), Vector2(1, 1), Vector2(1, 0), Vector2(0, 2)]
+				    # The function cmp(a, b) should return true if a &lt; b, here we compare the lengths.
```

Comment:
May I suggest "The function cmp(a, b) returns true if a is longer than b"? Or at least replace "should return" with "returns" to sound more final. Likewise below.

Reply:
The "should" is to indicate how the function should behave, indicating the requirement on it

Reply:
I don't think stating requirements makes sense here, since you implemented it below and it's just an example.

Reply:
You missed an "if" in the new wording.

Reply:
Sure, will correct